### ISSUE TITLE
Cannot find module 'eslint/lib/built-in-rules-index' error

### ISSUE THREAD

**Issue by salowenh** [aladdin-add/eslint-plugin#54]
* **eslint version:** 5.6.0
* **eslint-plugin-autofix version:** 0.0.0
* **eslint config:** 
* **command:** tsc && eslint --fix -c .eslintrc.json */**/*.ts
```
{
  ...
  "plugins": [
        "jsdoc",
        "autofix"
    ],
  "rules": {
    "autofix/sort-imports": "error"
  }
}
```

**Expected behavior** auto sort imports with no error

**Actual behavior** error:
```
Cannot find module 'eslint/lib/built-in-rules-index'
Require stack:
- /Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/rules.js
- /Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/configs/all.js
- /Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/index.js
- /Users/username/dev/username/node_modules/eslint/lib/config/plugins.js
- /Users/username/dev/username/node_modules/eslint/lib/config.js
- /Users/username/dev/username/node_modules/eslint/lib/cli-engine.js
- /Users/username/dev/username/node_modules/eslint/lib/cli.js
- /Users/username/dev/username/node_modules/eslint/bin/eslint.js
Error: Cannot find module 'eslint/lib/built-in-rules-index'
Require stack:
- /Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/rules.js
- /Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/configs/all.js
- /Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/index.js
- /Users/username/dev/username/node_modules/eslint/lib/config/plugins.js
- /Users/username/dev/username/node_modules/eslint/lib/config.js
- /Users/username/dev/username/node_modules/eslint/lib/cli-engine.js
- /Users/username/dev/username/node_modules/eslint/lib/cli.js
- /Users/username/dev/username/node_modules/eslint/bin/eslint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (/Users/username/dev/username/node_modules/v8-compile-cache/v8-compile-cache.js:161:20)
    at Object.<anonymous> (/Users/username/dev/username/node_modules/eslint-plugin-autofix/lib/rules.js:44:21)
    at Module._compile (/Users/username/dev/username/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:683:19)
```

**Comment by aladdin-add** [comment]
'eslint/lib/built-in-rules-index' was added in eslint v5.12.1. could you please  try `eslint>=5.12.1`?

**Comment by salowenh** [comment]
I've moved to use `sort-imports-es6-autofix/sort-imports-es6`
You should probably add prerequisites to the project 

Thank you for the quick reply.
